src/pages/journal/journal-components/journal-candidate/journal-candidate.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input } from '@angular/core';
+import { Name } from '../../../../common/domain/DJournal';
+
+@Component({
+  selector: 'journal-candidate',
+  templateUrl: 'journal-candidate.html'
+})
+export class JournalCandidateComponent {
+  @Input()
+  name: Name;
+
+  @Input()
+  testCategory: string;
+
+  @Input()
+  testComplete: boolean;
+
+  testCategoryDescription : string
+  constructor() {
+  }
+
+  ngOnInit() {
+    this.testCategoryDescription = this.getCategoryDescription();
+    this.testComplete = true; // not in json as yet... add when it is
+  }
+
+  getCategoryDescription(): string {
+    if (this.testCategory === 'B57mins') {
```

Comment:
Can that 'B57mins' constant come from a file which holds similar constant types as well please?

Would be good to have the other 2 constants in the same file. But they are less important.